.github/workflows/lint/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # Development
 FROM ubuntu:18.04
```

Comment:
From ubuntu:16.04 ?

Reply:
Since it works with 18.04 I'm inclined to leave it.  The OS version for the lint image is less important than the version of the tools installed for linting.

Reply:
agreed